packages/test/test-version-utils/src/test/versionUtils.spec.ts

```diff
@@ -3,134 +3,115 @@
+			checkRequestedVersionSatisfies("2.0.0-internal.7.0.0", -1, "^2.0.0-internal.6.0.0");
+			checkRequestedVersionSatisfies("2.0.0-internal.6.2.0", -2, "^2.0.0-internal.4.0.0");
+			checkRequestedVersionSatisfies("2.0.0-internal.6.2.0", -3, "^2.0.0-internal.3.0.0");
+		});
 
-		assert.strictEqual(
-			getRequestedRange("2.0.0-internal.6.0.0", -1),
-			">=2.0.0-internal.5.0.0 <2.0.0-internal.6.0.0",
-		);
-		assert.strictEqual(
-			getRequestedRange("2.0.0-internal.6.0.0", -2),
-			">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
-		);
-		assert.strictEqual(
-			getRequestedRange("2.0.0-internal.6.0.0", -3),
-			">=2.0.0-internal.3.0.0 <2.0.0-internal.4.0.0",
-		);
+		// TODO: What is the desired behavior here?
+		it.skip("bumping up internal releases to other internal releases", () => {
+			checkRequestedVersionSatisfies("2.0.0-internal.2.0.0", 1, "^2.0.0-internal.3.0.0");
+			checkRequestedVersionSatisfies("2.0.0-internal.2.1.1", 1, "^2.0.0-internal.3.0.0");
+			checkRequestedVersionSatisfies("2.0.0-internal.2.0.0", 2, "^2.0.0-internal.4.0.0");
+		});
 
-		// asserts for malformed major versions
-		assert.strictEqual(getRequestedRange("2.0.0", 0), "2.0.0");
-		assert.strictEqual(getRequestedRange("2.0.0", undefined), "2.0.0");
-		assert.throws(
-			() => getRequestedRange("-1.-2.-1", -1),
-			Error,
-			"TypeError: Invalid Version: -1.-2.-1",
-		);
-		assert.throws(
-			() => getRequestedRange("1.-2.-1", -1),
-			Error,
-			"TypeError: Invalid Version: 1.-2.-1",
-		);
-		assert.throws(
-			() => getRequestedRange("1.-2.-1", -1),
-			Error,
-			"TypeError: Invalid Version: 1.-2.-1",
-		);
-		assert.throws(
-			() => getRequestedRange("badString", -1),
-			Error,
-			"TypeError: Invalid Version: badString",
-		);
+		it("error cases for malformed versions", () => {
+			assert.strictEqual(getRequestedVersion("2.0.0", 0), "2.0.0");
+			assert.strictEqual(getRequestedVersion("2.0.0", undefined), "2.0.0");
+			assert.throws(
+				() => getRequestedVersion("-1.-2.-1", -1),
+				Error,
+				"TypeError: Invalid Version: -1.-2.-1",
+			);
+			assert.throws(
+				() => getRequestedVersion("1.-2.-1", -1),
+				Error,
+				"TypeError: Invalid Version: 1.-2.-1",
+			);
+			assert.throws(
+				() => getRequestedVersion("1.-2.-1", -1),
+				Error,
+				"TypeError: Invalid Version: 1.-2.-1",
+			);
+			assert.throws(
+				() => getRequestedVersion("badString", -1),
+				Error,
+				"TypeError: Invalid Version: badString",
+			);
+		});
 
-		// assert for minor bumps
-		assert.strictEqual(getRequestedRange("0.59.1000", -1), "^0.58.0-0");
-		assert.strictEqual(getRequestedRange("0.59.2000", -1), "^0.58.0-0");
-		assert.strictEqual(getRequestedRange("0.59.2000", -1), "^0.58.0-0");
+		it("bumping down public releases (minor)", () => {
+			checkRequestedVersionSatisfies("0.59.1000", -1, "^0.58.0");
+			checkRequestedVersionSatisfies("0.59.2000", -1, "^0.58.0");
+			checkRequestedVersionSatisfies("0.59.2000", -1, "^0.58.0");
+		});
```

Comment:
DOes it matter that the ranges are no longer pre-release (i.e. now they don't have the `-0`)?

Reply:
Should be fine since we now use `satisfies.satisfies()` to check if the version matches (instead of matching an exact version).